pom.xml

```diff
@@ -695,7 +695,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <argLine>@{argLine} -Djava.awt.headless=true</argLine>
-                    <excludedGroups>io.confluent.common.utils.IntegrationTest</excludedGroups>
+                    <excludedGroups>io.confluent.common.utils.IntegrationTest, IntegrationTest</excludedGroups>
```

Comment:
I'm not actually sure if this should be `io.confluent.common.utils.IntegrationTest | IntegrationTest` as I've seen both ways in documentation 
- https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html 
- https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit.html